src/envoy/token/token_subscriber_test.cc

```diff
@@ -340,6 +340,47 @@ TEST_F(TokenSubscriberTest, TokenSantizationCheckFails) {
   ASSERT_FALSE(init_ready_);
 }
 
+TEST_F(TokenSubscriberTest, TokenPastExpiryCheckFails) {
+  // Setup fake remote request.
+  Envoy::Http::RequestHeaderMapPtr req_headers(
+      new Envoy::Http::TestRequestHeaderMapImpl());
+  EXPECT_CALL(*info_, prepareRequest(token_url_))
+      .Times(1)
```

Comment:
It seems that all tests use the same test flow,  just wondering if we can use same code to share the flow. the parameters can be passed in.    I feel it will make test code more readable. 

This can be a separate PR.